src/isomorphic/devtools/ReactComponentTreeDevtool.js

```diff
@@ -19,6 +19,7 @@ var rootIDs = [];
 function updateTree(id, update) {
   if (!tree[id]) {
     tree[id] = {
+      source: null,
```

Comment:
I think storing element seems reasonable enough, but I believe React Devtools is set up to store its own tree so I'm not actually sure this would help it. Maybe I'm misremembering.